site/cds_rdm/inspire_harvester/transform_entry.py

```diff
@@ -64,6 +80,10 @@ def build(self):
             "access": self._access(),
         }
 
+        current_app.logger.info(
+            f"Building CDS-RDM entry record finished. RDM record: {rdm_record}."
+        )
+        current_app.logger.info(f"Errors: {self.errors}.")
```

Comment:
```suggestion
        current_app.logger.error(f"Errors: {self.errors}.")
```
plus the partial success report